Frontend/src/Components/PPC_MPPT/PPC_MPPT.js

```diff
@@ -34,43 +37,113 @@ export default function PPC_MPPT(props) {
                     <Spacer/>
                     <Flex w='45%' direction='column'>
                         <MPPT_CELL
-                            label='Charge'
+                            label='Tracking'
                             isCurrMode={props.data?.mppt_mode[0]}
                         />
                         <Spacer/>
                         <MPPT_CELL
-                            label='MPPT'
+                            label='Current Control'
                             isCurrMode={!props.data?.mppt_mode[0]}
                         />
                     </Flex>
                 </Flex>
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 1 Temp'
-                    data={props.data?.string1_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string1_temp.MIN}
-                    max={CONSTANTS.string1_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string2_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string2_temp.MIN}
-                    max={CONSTANTS.string2_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string3_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string3_temp.MIN}
-                    max={CONSTANTS.string3_temp.MAX}
-                />
+                <Grid
+                    py='1rem'
+                    templateColumns="repeat(4, 1fr)"
+                    templateRows="repeat(4, 1fr)"
+                    gap={0.5}
+
+                >
+                    <Box px='1.4rem'></Box>
+                    <GridItem bg='gray' color='black' fontWeight='bold' height='20px' fontSize={10} alignItems='center' display='flex'>Temp(ºC)</GridItem>
+                    <GridItem bg='gray' color='black' fontWeight='bold' height='20px' fontSize={10} alignItems='center' display='flex'>Voltage(V)</GridItem>
+                    <GridItem bg='gray' color='black' fontWeight='bold' height='20px' fontSize={10} alignItems='center' display='flex'>Current(A)</GridItem>
+                    <Box alignItems='right' fontSize={12}>String 1</Box>
```

Comment:
I don't think the first Box is needed, as you can define the location of each GridItem in terms of row and column instead of adding a placeholder element.

For the second Box, consider changing that to a GridItem

Reply:
I changed the second box to a gridItem but the Box at the first makes the alignment a lot easier because the top left box needs to be empty so the box makes it easy to do that. When I get rid of the first box, it moves all the labels to the opposite side so keeping that might make the implementation easier. Not sure what you think about this but I did change the second box to a gridItem

Reply:
actually you were right the box is not needed, I was just using it for the top left blank part of the grid but I just used a spacer instead, so this comment should be fixed

Reply:
@VIkranthMandadi, regarding the Box that you changed to a Spacer, I think that component could be removed entirely. You can specify the location of individual GridItems within a Grid using the `rowStart` and `colStart` props. By specifying the location of every GridItem, you can leave the unspecified location/cell empty without having to add a placeholder prop to fill that cell. If this gives you trouble though, it would still be preferable to use a GridItem for the empty cell rather than a Spacer.